util/pkg/architectures/architectures.go

```diff
@@ -41,7 +42,13 @@ func FindArchitecture() (Architecture, error) {
 	}
 }
 
-func GetSupprted() []Architecture {
+func GetSupported() []Architecture {
+	// Force support for AMD64 only if env var is set
+	arch := os.Getenv("KOPS_ARCH_AMD64")
+	if arch != "" {
+		return []Architecture{ArchitectureAmd64}
+	}
```

Comment:
Nice! Appreciate the fix!

Any idea whether we need to update the documentation?

Reply:
Np. This is for internal use at the moment, so I don't see much value in adding docs for it.

Reply:
I wonder if we could setup this env var so its a bit more flexible in the future if we do ever support additional architectures? Perhaps move the architecture into the env var value:

`KOPS_ARCH=amd64`
`KOPS_ARCH=amd64,arm64`

Reply:
I think in the future we may want to make it a spec option or even a create cluster flag.
At the moment, there is very little value to allow various values.

This function really should be used for anything cpu arch related in case there will be something like that, and it touches cni plugins and maybe other types of assets.

If we ever want to, we could add also `KOPS_ARCH_ARM64` if we only want ARM64 support, but I don't have a better idea either.

Reply:
Another dump question. :)

I assume it was too late to rely on `FindArchitecture()` to provide the correct arch at runtime probably because we want to download binary/images upfront to save time?

Reply:
You would have to know what arch is on each node. That is not possible or unreliable when the cluster is created.
We only retrieve the hashes, not the binaries. Each node decides on its own what arch it needs.